examples/models/core/enc_dec/convert_checkpoint.py

```
@@ -1479,6 +1503,119 @@ def get_model(args):
+
+            processor = get_processor()
+            model = VisionEncoderDecoderModel.from_pretrained(
+                "facebook/nougat-base")
+            model.encoder = RadioWithNeck()
+            model.decoder.resize_token_embeddings(len(processor.tokenizer),
+                                                  pad_to_multiple_of=64)
+            model.config.decoder_start_token_id = processor.tokenizer.eos_token_id  # 2
+            model.config.pad_token_id = processor.tokenizer.pad_token_id  # 1
+            from transformers.models.mbart.modeling_mbart import \
+                MBartLearnedPositionalEmbedding
+            device, d_model = model.device, model.config.decoder.d_model
```

Ruff (0.12.2)

Local variable `device` is assigned to but never used

Remove assignment to unused variable `device`

(F841)